release/e2e.py

```diff
@@ -846,17 +846,18 @@ def create_and_wait_for_session(
     start_wait = time.time()
     next_report = start_wait + REPORT_S
     while not completed:
+        session_operation_response = sdk.get_session_operation(
+            sop_id, _request_timeout=30)
+        session_operation = session_operation_response.result
+        completed = session_operation.completed
+
```

Comment:
This move will actually not alter sleeping behavior (as the `not completed` check will only be invoked after the sleep). We should probably just move the `time.sleep(1)` to the top here or just 

```
if not completed:
   time.sleep(1)
```

Reply:
oh yeah, good point.
I was too focused on making sure we don't wait a 1 sec before checking the returned result.
this is better.